spec/dummy/app/avo/resources/post_resource.rb

```diff
@@ -34,7 +34,7 @@ class PostResource < Avo::BaseResource
   end
   field :user, as: :belongs_to, placeholder: "—"
   field :status, as: :select, enum: ::Post.statuses, display_value: false
-  field :comments, as: :has_many
+  field :comments, as: :has_many, use_resource: PhotoCommentResource, searchable: true
```

Comment:
Let's make this unsearchable please.
```suggestion
  field :comments, as: :has_many, use_resource: PhotoCommentResource
```

Reply:
agree